site/content/zh-CN/docs/reference/kubectl-kueue/commands/kueuectl_version/_index.md

```diff
@@ -5,30 +5,24 @@ auto_generated: true
 no_list: true
 ---
 
-<!--
-The file is auto-generated from the Go source code of the component using the
-[generator](https://github.com/kubernetes-sigs/kueue/tree/main/cmd/kueuectl-docs).
--->
+## 概要
 
-## Synopsis
-
-
-Prints the client version and the kueue controller manager image, if installed
+打印客户端版本和 kueue 控制器管理器镜像（如果已安装）
 
 ```
 kueuectl version
 ```
 
 
-## Examples
+## 示例
 
 ```
-  # Prints the client version and the kueue controller manager image, if installed
+  # 打印客户端版本和 kueue 控制器管理器镜像（如果已安装）
```

Comment:
```suggestion
  # 打印客户端版本和 Kueue 控制器管理器镜像（如果已安装）
```
We can use `Kueue` after localization, to keep consistent with the logo:
https://kueue.sigs.k8s.io/